templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -53,7 +53,9 @@
   {
```

Comment:
<p>I think maybe something like "Now we want to examine the number of each cell type assigned to each kmeans neighborhood cluster (z-scored for each cell type)." would be more clear?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>